Integrations/integration-Okta.yml

```diff
@@ -337,6 +474,15 @@ script:
         for usernames that contains a '/' character
     - name: userid
       description: User ID of requested user
+    - name: verbose
+      auto: PREDEFINED
+      predefined:
+      - "true"
+      - "false"
+      description: Print all details of user
+    outputs:
+    - contextPath: Account
```

Comment:
Need to specify the leafs (fields) for the account with a separate description for each - otherwise not usable in playbook.